example-java/src/main/java/com/softwaremill/kmq/example/standalone/StandaloneConfig.java

```diff
@@ -6,10 +6,10 @@
 import java.time.Duration;
 
 class StandaloneConfig {
-    static final KmqConfig KMQ_CONFIG = new KmqConfig("queue", "markers", "kmq_client",
-            "kmq_redelivery", Duration.ofSeconds(90).toMillis(), 1000);
+    static final KmqConfig KMQ_CONFIG = new KmqConfig("queue", "markers", "kmq_client", "kmq_redelivery",
+            Duration.ofSeconds(90).toMillis(), 1000);
 
-    /* EXAMPLE with extraConfig : SSL Encryption & SSL Authentication
+    /* TODO: EXAMPLE with extraConfig : SSL Encryption & SSL Authentication
```

Comment:
is there something to do here?

Reply:
No, I was just my first reaction when I saw commented-out code :) 
On the second thought it's okay in an example.